requirements/tox.in

```diff
@@ -0,0 +1,17 @@
+-c base.in
```

Comment:
These are for specifying [constraint files](https://pip.pypa.io/en/latest/user_guide/#constraints-files).  They are used for adding contraints to the versions of things that are pinned, without explicitly pulling in those dependencies. I had this because there were overlapping dependencies, which we had more stringent constraints on than would be generated by this file.

On reflection, I'm not sure that there is a good reason to not just make this a superset of the dependencies (and just use `-r test.in` here, and then just have `-r requirements/tox.txt` in `tox.ini`.